client/scss/application.scss

```diff
@@ -5,6 +5,7 @@
 @import 'utilities/variables';
 @import 'utilities/mixins';
 @import 'utilities/functions';
+@import 'utilities/classes';
```

Comment:
That's a very vague name? Aren't all of these classes?
Annoying these are utils, but that's the parent dir (which might be the problem).

What about `generic` or `universal`?